lib/wallet/rpc.js

```diff
@@ -2039,41 +2036,6 @@ class RPC extends RPCBase {
     return {name, force, account};
   }
 
-  async sendOpenIdempotent(args, help) {
```

Comment:
We are completely doing away with these RPC methods, right? Not supporting adding new ones like we are for routes.

Reply:
Yes, all interaction between namebase app and wallet will be using http methods from now on. We wont be adding new routes for RPC methods and refactor existing ones to use http ones instead